src/System.Private.CoreLib/shared/System/IO/TextWriter.cs

```diff
@@ -146,7 +146,7 @@ public virtual void Write(char value)
         // Write(char) for each of the characters in the character array.
         // If the character array is null, nothing is written.
         //
-        public virtual void Write(char[] buffer)
+        public virtual void Write(char[]? buffer)
```

Comment:
Feels weird that this one doesn't throw when buffer is null but the other overload does.

Reply:
Actually now that I think it more it makes sense, why would you pass a null buffer and index and a count all together.